deployment/data-streams/changeset/verifier-proxy/v0.5.0/call_unset_verifier.go

```diff
@@ -0,0 +1,90 @@
+package v0_5_0
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	mcmslib "github.com/smartcontractkit/mcms"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/types"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/utils/mcmsutil"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/utils/txutil"
+)
+
+// UnsetVerifierChangeset sets the active FeeManager contract on the proxy contract
+var UnsetVerifierChangeset deployment.ChangeSetV2[VerifierProxyUnsetVerifierConfig] = &verifierProxyUnsetVerifier{}
+
+type verifierProxyUnsetVerifier struct{}
+
+type VerifierProxyUnsetVerifierConfig struct {
+	ConfigPerChain map[uint64]UnsetVerifierConfig
```

Comment:
I was using 
`ConfigPerChain map[uint64][]UnsetVerifierConfig` (slice of UnsetVerifierConfig) in other changesets.
Do you think we should allow just 1 contract update per changeset to make things simpler?

Reply:
I see no reason to not allow the flexibility so I've added it